src/one-time-token/controller.ts

```diff
@@ -0,0 +1,39 @@
+import Controller from '@curveball/controller';
+import { Context } from '@curveball/core';
+import * as privilegeService from '../privilege/service';
+import { Forbidden, NotFound } from '@curveball/http-errors';
+import * as userService from '../user/service';
+import { User } from '../user/types';
+import { createToken } from '../reset-password/service';
+
+class OneTimeTokenController extends Controller {
+
+  async post(ctx: Context<any>) {
+
+    if (!await privilegeService.hasPrivilege(ctx, 'admin')) {
+      throw new Forbidden('Only users with the "admin" privilege can create new users');
```

Comment:
wrong error message.

It might also be nice at this point to start making more specific privileges for specific tasks? We can do that in another ticket though.